pretraining/utils/train_utils.py

```diff
@@ -1,14 +1,16 @@
 import os
-import time
 
-import torch.cuda.nccl as nccl
-import torch.distributed as dist
 
 try:
     import packaging.version
 except ImportError:
-    from pkg_resources import packaging
+    from pkg_resources import packaging  # type: ignore
 
+import time
+
+import torch.cuda.nccl as nccl
+import torch.distributed as dist
+from packaging import version
```

Comment:
this is the import that was in a try block above. if it fails above it would fail here too.